statsmodels/stats/proportion.py

```diff
@@ -123,6 +123,183 @@ def func(qi):
+    intervals closer to the desired significance level), but is valid when the
+    number of categories is large. In their paper, Sison & Glaz demo their
+    method with at least 7 categories, so ``len(counts) >= 7`` can be used as a
+    rule of thumb.
+
+    Aside from the original sources ([1]_, [2]_, and [3]_), the implementation
+    uses the formulas (though not the code) presented in [4]_ and [5]_.
+
+    References
+    ----------
+    .. [1] Levin, Bruce, "A representation for multinomial cumulative
+           distribution functions," The Annals of Statistics, Vol. 9, No. 5,
+           1981, pp. 1123-1126.
+
+    .. [2] Goodman, L.A., "On simultaneous confidence intervals for multinomial
+           proportions," Technometrics, Vol. 7, No. 2, 1965, pp. 247-254.
+
+    .. [3] Sison, Cristina P., and Joseph Glaz, "Simultaneous Confidence
+           Intervals and Sample Size Determination for Multinomial
+           Proportions," Journal of the American Statistical Association,
+           Vol. 90, No. 429, 1995, pp. 366-369.
+
+    .. [4] May, Warren L., and William D. Johnson, "A SAS® macro for
+           constructing simultaneous confidence intervals  for multinomial
+           proportions," Computer methods and programs in Biomedicine, Vol. 53,
+           No. 3, 1997, pp. 153-162.
+
+    .. [5] May, Warren L., and William D. Johnson, "Constructing two-sided
+           simultaneous confidence intervals for multinomial proportions for
+           small counts in a large number of cells," Journal of Statistical
+           Software, Vol. 5, No. 6, 2000, pp. 1-24.
+    '''
+    if alpha <= 0 or alpha >= 1:
+        raise ValueError('alpha must be in (0, 1), bounds excluded')
+    counts = np.array(counts, dtype=np.float)
+    if (counts < 0).any():
+        raise ValueError('counts must be >= 0')
+    n = counts.sum()
+    k = len(counts)
+    estimated_parameters = counts / n
+    if method == 'goodman':
+        chi2 = stats.chi2.ppf(1 - alpha / k, 1)
+        delta = chi2 ** 2 + (4 * n * estimated_parameters *
+                             chi2 * (1 - estimated_parameters))
+        region = ((2 * n * estimated_parameters + chi2 +
+                   np.array([- np.sqrt(delta), np.sqrt(delta)])) /
+                  (2 * (chi2 + n))).T
+    elif method == 'sison_glaz':
+        # Define a few functions we'll use a lot.
+        def poisson_interval(interval, l):
+            """Compute P(b <= Z <= a) where Z ~ Poisson(l) and
+            `interval = (b, a)`."""
+            b, a = interval
+            return stats.poisson.cdf(a, l) - stats.poisson.cdf(b - 1, l)
+
+        def truncated_poisson_factorial_moment(interval, r, l):
+            """Compute mu_r, the r-th factorial moment of a poisson random
+            variable of parameter `l` truncated to `interval = (b, a)`."""
+            b, a = interval
+            return l ** r * (1 - ((poisson_interval((a - r + 1, a), l) -
```

Comment:
Right, changed to `p`. (Although I remember this problems appears pretty often, GitHub's rendering on my browser is particularly bad at this (might be my available fonts). In my terminal `1` and `l` look really different.)